nymea-app/ui/mainviews/DevicesPageDelegate.qml

```diff
@@ -18,7 +18,7 @@ Item {
             anchors.verticalCenterOffset: -app.iconSize
             spacing: app.margins
             ColorIcon {
-                height: app.iconSize * 2
+                height: app.iconSize * 1.3
```

Comment:
if you're changing the icon size here and in FavoritesView, shouldn't it be changed in ScenesView as well so we have all 3 main pages consistent?